prelude/align.md

```diff
@@ -23,5 +23,26 @@ impl Align {
     pub fn bytes(self) -> BigInt {
         self.raw
     }
+
+    /// Computes the best alignment possible for the given offset
+    /// (the largest power of two that the offset is a multiple of).
+    ///
+    /// N.B., for an offset of `0`, this returns `2^64`.
```

Comment:
Do you prefer another handling of the `offset == zero` case?
This is how rustc does it, but `2^64` is somewhat arbitrary for us - as we are working with BigInt.

Reply:
This should probably return `Option<Align>`, and return `None` if the offset is zero.